### ISSUE TITLE
Full size product images

### ISSUE THREAD

**Issue by bazaarbroz** [OpenBazaar/OpenBazaar-Client#841]
Hi there!

I've been experimenting with the test version and have found that when uploading a larger image, the images are resized/compressed -  so a large pic loses all of its finer detail - is there a way that a user can click to see a larger size (or ideally, the original size?)

Also, the aspect ratio/orientation of the product image preview thumbnail seems to be fixed - I was trying to upload a portrait style image but it was only displaying the middle section of it because the thumbnail was not the same shape

Basically I think it would be great if there could be a bit more flexibility with the product images - it can be very useful for showing off different products correctly. I have high res product pictures that will be wasted on OB!

thanks for all the work done so far!

**Comment by jjeffryes** [comment]
Currently, only a single image version is saved for each image, so to conserve bandwidth it is saved at a relatively low size.

In future iterations we will save multiple sizes, so thumbnails and preview images are different files from the full size images. That will allow the full size images to be higher resolution.

**Comment by jjeffryes** [comment]
I'm going to close this, as it's already on our roadmap.
